packages/ui/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "lit": "2.8.0",
+    "motion": "10.16.2",
```

Comment:
Moved to new place, but package lock was not uodated, run `npm run clean; npm install` to apply this change